R/analyse.R

```diff
@@ -13,7 +13,17 @@
 #'
 #' @param config either a path to a config.yml file or a list of parameters
 #'
-#' @return NULL
+#' @return
+#'
+#' `analyse()` returns `NULL` invisibly. It is called for its side-effect of
+#' creating the outputs of the standard PACTA analysis and the outputs of the
+#' net aggregated alignment metric in your `output/analysis` directory.
```

Comment:
nit: to be super pedantic, the output directory is whatever the user specifies in `dir_analysis` in the `config.yml`..

Reply:
hmmmm, initially I was thinking since this is text the CRAN folks will likely review closely, let's not make it too complicated, but maybe it's not too complicated and more correct to say...
> `analyse()` returns `NULL` invisibly. It is called for its side-effect of creating the outputs of the standard PACTA analysis and the outputs of the net aggregated alignment metric in the directory specified by `dir_analysis` in your config file.

does that sound good/right @jacobvjk?

Reply:
that sounds more correct, but admittedly I am not sure what the people at CRAN would prefer. I think we just try one and worst case we will have to adjust it

Reply:
ok, I'll go with "more correct" for now... changes incoming